DFC.Digital/DFC.Digital.Web.Sitefinity.DfcSearchModule/Extensions/HelperExtensions.cs

```diff
@@ -17,6 +17,13 @@ internal static IEnumerable<JobProfileIndex> ConvertToJobProfileIndex(this IEnum
             List<Task> salaryPopulation = new List<Task>();
             foreach (var item in documents)
```

Comment:
Change this line to 

```
var betaDocuments = documents.Where(d => Convert.ToBoolean(d.GetValue(nameof(JobProfile.IsImported)) ?? false) == false);
foreach (var item in betaDocuments)...
```

Reply:
I have changed it to 
           foreach (var item in documents)
            {
                bool result;
                bool itemIsImported = bool.TryParse(item.GetValue(nameof(JobProfileIndex.IsImported))?.ToString(), out result) ? result : false;

and have placed it inside the foreach loop.

Reply:
OK. I've changed it exactly as recommended.